### PR TITLE
Quest solutions

### PR DESCRIPTION
**Pull request by kjhenner**

### CODE REVIEW

File: Quest_Guide/quests/modules_quest.md [hunk 1]
```diff
@@ -177,6 +190,15 @@ Puppet's built-in fileserver.
     cp ~/.vimrc vimrc/files/vimrc
 	
 {% task 5 %}
+- execute: vim /etc/puppetlabs/puppet/environments/production/modules/vimrc/files/vimrc
```
Comment (casharma) [inline Quest_Guide/quests/modules_quest.md]: do missing "---" matter? There are a few missing starting from here.

File: quest_tool/.testing/spec/localhost/manifests_classes_spec.rb [hunk 2]
```diff
@@ -65,6 +65,8 @@
 # Task 9
 describe "The cowsayings/test/init.pp test manifest" do
   it "should be applied" do
-    file('/root/.bash_history').should contain 'puppet apply cowsayings/tests/init.pp'
+    file('/etc/puppetlabs/puppet/environments/production/modules/cowsayings.tests.init.pp').should be_file
```
Comment (casharma) [inline quest_tool/.testing/spec/localhost/manifests_classes_spec.rb]: cowsayings/tests/init.pp not .tests.init.pp

**Comment by kjhenner** [comment]
Thanks, Carthik. I'll fix these and run through everything one more time on a fresh VM.

**Comment by kjhenner** [comment]
Made changes per comments above and successfully ran through solutions for each quest on a new VM.

File: Quest_Guide/quests/modules_quest.md [hunk 3]
```diff
@@ -281,6 +318,8 @@ include vimrc
 {% endhighlight %}
 
 {% task 8 %}
+- execute: puppet apply /etc/puppetlabs/puppet/environments/production/modules/vimrc/tests/init.pp
```
Comment (casharma) [inline Quest_Guide/quests/modules_quest.md]: ---

File: Quest_Guide/quests/power_of_puppet.md [hunk 4]
```diff
@@ -187,6 +193,10 @@ necessary to bring the node into the line with the state described by the
 catalog.
 
 {% task 3 %}
+- execute: |
```
Comment (casharma) [inline Quest_Guide/quests/power_of_puppet.md]: ---

File: Quest_Guide/quests/variables_and_parameters.md [hunk 5]
```diff
@@ -131,6 +135,26 @@ Now create an `web` directory:
     mkdir web/{manifests,tests}
 
 {% task 2 %}
+- file: /etc/puppetlabs/puppet/environments/production/modules/web/manifests/init.pp
```
Comment (casharma) [inline Quest_Guide/quests/variables_and_parameters.md]: ---

File: quest_tool/.testing/spec/localhost/manifests_classes_spec.rb [hunk 6]
```diff
@@ -65,6 +65,8 @@
 # Task 9
 describe "The cowsayings/test/init.pp test manifest" do
```
Comment (casharma) [inline quest_tool/.testing/spec/localhost/manifests_classes_spec.rb]: tests, not test

File: quest_tool/.testing/spec/localhost/manifests_classes_spec.rb [hunk 7]
```diff
@@ -65,6 +65,8 @@
...
   it "should be applied" do
-    file('/root/.bash_history').should contain 'puppet apply cowsayings/tests/init.pp'
+    file('/etc/puppetlabs/puppet/environments/production/modules/cowsayings/tests.init.pp').should be_file
```
Comment (casharma) [inline quest_tool/.testing/spec/localhost/manifests_classes_spec.rb]: tests/init.pp not tests.init.pp

**Comment by kjhenner** [comment]
Fixed missing yaml separators and a fews filepath typos in spec tests

**Pull request merged**